src/runtime_impl.cpp

```diff
@@ -145,7 +141,7 @@ namespace hpx {
         thread_manager_(new hpx::threads::threadmanager_impl<
             SchedulingPolicy, NotificationPolicy>(
                 timer_pool_, scheduler_, notifier_, num_threads)),
-        parcel_handler_(thread_manager_.get(),
+        parcel_handler_(rtcfg, agas_client_, thread_manager_.get(),
             new parcelset::policies::global_parcelhandler_queue,
```

Comment:
I believe this is the result of a bad merge as I remember to have fixed that already.